gossip/src/cluster_info.rs

```diff
@@ -3071,7 +3079,7 @@ impl Node {
             )
             .unwrap();
         let tpu_quic =
-            bind_more_with_config(tpu_quic, QUIC_ENDPOINTS, quic_config.clone()).unwrap();
+            bind_more_with_config(tpu_quic, DEFAULT_QUIC_ENDPOINTS, quic_config.clone()).unwrap();
```

Comment:
What if we decided to change DEFAULT_QUIC_ENDPOINTS to a different value than 1?

Reply:
It should work with DEFAULT_QUIC_ENDPOINTS > 1. This code is used only for testing.

Reply:
Maybe we should set this to 1 then, rather than relying on DEFAULT_QUIC_ENDPOINTS, since the semantics of  new_single_bind implies 1, even if DEFAULT_QUIC_ENDPOINTS, changes.

Reply:
You are raising a good point. But this is an orthogonal issue -- not caused by this PR. It is the implementation in the PR where we did the multiquic change. Also -- I yet to understand what that "single" mean in this context. Anyway, I would like it to be addressed a separate PR. @ryleung-solana